client/components/domains/domain-suggestion/style.scss

```diff
@@ -16,7 +16,7 @@
 	&.is-clickable {
 		cursor: pointer;
 
-		&:hover {
+		&:hover, &:focus {
```

Comment:
Minor point, but let's move this to the next line:
```
	&:hover,
	&:focus {
```
It makes it a bit easier not to misread the selector here.